components/maincontroller.js

```diff
@@ -273,6 +277,9 @@
             if (!isPlaying()) {
                 embyActions.displayUserInfo($scope, data.serverAddress, data.accessToken, data.userId);
             }
+            else {
+                embyActions.reportPlaybackProgress($scope, getReportingParams($scope), true, reportEventType);
+            }
```

Comment:
when a client reconnects it sends the 'Identify' command, and if something is currently playing (or paused) report playbcack progress back so the UI in the app is updated correctly and shows the currently playing item as playing.